src/components/app/ProfileSharing.js

```diff
@@ -277,6 +291,16 @@ class ProfileSharingCompositeButton extends PureComponent<
               onOkButtonClick={this._attemptToShare}
             >
               <PrivacyNotice />
+              <p>
+                <label>
+                  <input
+                    type="checkbox"
+                    onChange={this._toggleShareNetworkUrls}
+                    checked={this.state.shareNetworkUrls}
+                  />
+                  Share the URLs of all network requests
+                </label>
+              </p>
```

Comment:
We could use some CSS to center the text with the checkbox:
![screen shot 2018-06-22 at 14 27 04](https://user-images.githubusercontent.com/454175/41776535-6904e5d8-7628-11e8-9edb-a026386e6a42.png)